generate_wordstr_box.py

```diff
@@ -44,6 +48,10 @@ def trim(im):
 # create WordStr line boxes for Indic & RTL
 for line in lines:
     line = unicodedata.normalize('NFC', line.strip())
+    if args.rtl:
+        # FIXME: This should not be necessary. Compare with e.g. kraken
+        line = line.translate(str.maketrans("()[]{}»«><", ")(][}{«»<>"))
```

Comment:
According to https://github.com/tesseract-ocr/tesstrain/pull/137#issuecomment-572041875, the previous two lines should be removed.